jmh/src/main/scala/filodb.jmh/QueryHiCardInMemoryBenchmark.scala

```diff
@@ -36,9 +37,9 @@ class QueryLeafScanInMemoryBenchmark extends StrictLogging {
   import NodeClusterActor._
 
   val numShards = 1
-  val numSeries = 200
-  val samplesDuration = 2.hours
-  val numSamples = numSeries * (samplesDuration / 10.seconds).toInt
+  val numSeries = 8000   // NOTE: num of time series queried is this / 4
+  val samplesDuration = 15.minutes
+  val numSamples = (samplesDuration / 10.seconds).toInt   // # samples PER TIME SERIES
```

Comment:
Please add additional benchmarks for longer query intervals with very significantly larger number of steps.